XU-TechnicalPaper.md

```diff
@@ -25,163 +25,176 @@
+Each participating exchange runs its own XUD instance, which opens payment channels on each chain it desires to enable trading for. This means, if for example an exchange wants to enable its users to trade bitcoin and litecoin through XUD, it has to open a payment channel on the litecoin blockchain & on the bitcoin blockchain to at least one other exchange in the union. Payment channels in XU are meant to be kept open long-term. Since routing is and will be relatively unreliable for larger amounts in the foreseeable future, the current design encourages larger, direct channels between exchanges. Currently, lnd's maximum channel size is temporarily limited to 0.16777216 BTC. This can be lifted using a so-called [wumbo](https://github.com/lightningnetwork/lightning-rfc/wiki/Lightning-Specification-1.1-Proposal-States)-bit. To achieve long-term scalability, even with routing, we will rely on multiple channels, which carry out partial transfers of one atomic payment as proposed in the [AMP protocol by Laolu and Conner](https://lists.linuxfoundation.org/pipermail/lightning-dev/2018-February/000993.html).
 
-Also, for payment channels to fully work, transaction malleability has to be fixed on the underlying chain. This happened with the activation of SegWit on Bitcoin, Litecoin and some other Bitcoin-based chains like Vertcoin in 2017; Ethereum’s transaction malleability is already fixed since the Homestead release in 2016.
+In general, all BIP 199 compatible payment channels can be supported by XUD directly, other chains may implement HTLC swaps via smart contracts. The payment channel standard on bitcoin and litecoin is called the [lightning network](https://lightning.network/) and compatible implementations are maintained by the companies [ACINQ](https://github.com/ACINQ/eclair), [Lightning Labs](https://lightning.engineering/) and [Blockstream](https://blockstream.com/) which are all [intercompatible] by following the so-called [BOLT](https://github.com/lightningnetwork/lightning-rfc/blob/master/00-introduction.md) specifications. Ethereum’s most advanced payment channel technology is the ‘[Raiden Network](https://raiden.network/)’ and is mainly maintained by the company [Brainbot](http://www.brainbot.com/).
 
+**XUD, using LND and Raiden, represents a sub-graph of the lightning & raiden network. It's a primary goal to remain fully compatible with the [lightning BOLT specifications (BOLT)](https://github.com/lightningnetwork/lightning-rfc/blob/master/00-introduction.md) and with the specifications of the raiden network.** XUD acts as a wrapper for the payment channel layer and encapsulates lightning and raiden daemons with as little modifications as possible.
```

Comment:
Is it a sub graph? It would just be part of those networks right? Also instead of "It's a primary goal to..." you can say "Its primary goal is to..." where "it" I'm assuming is xud.

Reply:
Sub-graph is a part. But yeah I just wrote part, no need to be complicated.

> Its primary goal is to..." where "it" I'm assuming is xud.

Solved it with "The primary goal is to remain fully compatible..."